### PR TITLE
Landingpage

### PR DESCRIPTION
**Pull request by Somidh**

### CODE REVIEW

File: components/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { FC } from "react";
+
+const Navbar: FC = () => {
+  return (
+    <div className="flex items-center justify-between py-10">
+      <h1 className="text-[32px] font-bold">Earcandy</h1>
+
+      <ul className="flex items-center gap-16 text-xl font-bold">
```
Comment (Aadarsh805) [inline components/Navbar.tsx:8]: make these links

File: components/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        <li>Discover</li>
+        <li>Trending</li>
+        <li>Authors</li>
+        <li>Contact</li>
+      </ul>
+
+      <button className="text-[18px] text-white bg-black rounded-full py-[11px] px-16 font-bold">
```
Comment (Aadarsh805) [inline components/Navbar.tsx:15]: fix color here

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
add that sidebar as well, use `menu` that sagar created or if you have already created that sidebar, and is closer to design that sagar's then use yours in explore page as well, removing sagar's

File: components/Navbar.tsx (hunk 2 above)
Comment (Somidh) [reply components/Navbar.tsx:15]: what color to fix?

File: components/Navbar.tsx (hunk 1 above)
Comment (Somidh) [reply components/Navbar.tsx:8]: okay

Comment (Somidh) [reply components/Navbar.tsx:8]: > add that sidebar as well, use `menu` that sagar created or if you have already created that sidebar, and is closer to design that sagar's then use yours in explore page as well, removing sagar's

Okay for now i am adding mine sidebar, we can remove it later if sagar's one look better okay?

File: components/Navbar.tsx [hunk 3]
```diff
@@ -1,18 +1,49 @@
...
+            className={`text-xl font-bold ${noto_serif.className}`}
+          >
+            {item.name}
+          </Link>
+        ))}
       </ul>
+    
 
-      <button className="text-[18px] text-white bg-black rounded-full py-[11px] px-16 font-bold">
+      <button className={`text-[18px] text-white bg-[#303933] rounded-full py-[11px] px-16 font-bold ${noto_serif.className}`}>
```
Comment (Aadarsh805) [inline components/Navbar.tsx:46]: `bg-primary` check tailwind.config

File: pages/index.tsx [hunk 4]
```diff
@@ -47,10 +61,10 @@ const Home: FC = () => {
         <Image
           src={star}
           alt="star_img"
-          className="absolute bottom-0 left-[7em] w-8"
+          className="absolute bottom-0 left-[26em] w-8"
         />
         <Image src={star} alt="star_img" className="absolute top-0 right-80" />
-        <h1 className="">
+        <h1 className={`text-[65px] font-bold ${noto_serif.className}`}>
```
Comment (Aadarsh805) [inline pages/index.tsx:67]: this also in tailwind.config, use `font-noto`
also add playfair and open sans to tailwind config

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: components/Navbar.tsx (hunk 2 above)
Comment (Aadarsh805) [reply components/Navbar.tsx]: button bg not black in design

File: pages/index.tsx (hunk 4 above)
Comment (Somidh) [reply pages/index.tsx:67]: so should i remove the font.ts file?

**Pull request merged**